echidna/test/test_spectra.py

```diff
@@ -91,8 +91,12 @@ def test_scale(self):
             radius = random.uniform(radial_low, radial_high)
             test_spectra.fill(energy_mc=energy, radial_mc=radius)
         self.assertTrue(test_spectra.sum() == test_decays)
-        count = 150
+        count = 150  # int
+        test_spectra.scale(count)
+        count = 110  # int --> int(110) / int(150) = 0
         test_spectra.scale(count)
+        # Check sum != 0.0
+        self.assertNotEqual(test_spectra.sum(), 0.0)
```

Comment:
@jwaterfield, I've removed the other test I added, but I think this one would still be useful to keep in.